src/styles/components/_related-components.scss

```diff
@@ -84,5 +89,13 @@
         outline: 1px solid var(--accent-color);
       }
     }
+
+    @media (prefers-reduced-motion: reduce) {
+      &:hover,
+      &:focus {
+        outline: 1px solid var(--foreground-color);
```

Comment:
let's bump this to 2px to meet a11y standards

Reply:
Oh yes please! I don't know where I got the idea that outlines could only be 1px...but yes! This looks way better. 
https://github.com/sparkbox/accessible-components/pull/89/commits/d3aae7dedbb9362f058ee30fe4e6be5d0598c269